doc/source/structures/orbits/orbit.rst

```diff
@@ -197,10 +197,10 @@ Structure
     :type: :ref:`scalar <scalar>`
     :access: Get only
 
-    `mean anomaly`_  in degrees. Even though orbital parameters are
+    `mean anomaly`_  in degrees at orbit epoch. Even though orbital parameters are
     traditionally done in radians, in keeping with the kOS standard
     of making everything into degrees, they are given as degrees by
-    kOS.
+    kOS. Orbit epoch is a complex term, refer to `#1665<https://github.com/KSP-KOS/KOS/issues/1665>`_ for details. 
```

Comment:
I think it's just better than nothing. Might be reasonable to just notice that orbit epoch is not the same that the universe epoch. A link to KSP docs could be also helpful and better than to github issue.

I think something has to be done here. I'd lost a couple hours trying to catch a bug related to this.